packages/node_modules/@webex/plugin-meetings/sample/app.js

```diff
@@ -589,13 +603,33 @@ document.getElementById('join').addEventListener('click', () => {
     clearInterval(videoRenderInterval);
   }, 15000);
   meeting.members.on('members:created', (members) => {
-    document.getElementById('participants-collection').innerHTML = `MEMBERS INFO: 
-    selfList-${getParticipantString(members.selfList)}-
-    hostList-${getParticipantString(members.hostList)}-
-    inLobbyList-${getParticipantString(members.inLobbyList)}-
-    inMeetingList-${getParticipantString(members.inMeetingList)}-
-    notInMeetingList-${getParticipantString(members.notInMeetingList)}-
-    `;
+    let memberString = '';
+    Object.keys(members).forEach((key) => {
+      const member = members[key];
+      memberString += `Member: ${getParticipantNameString(member.participant)} Values: ${getParticipantValueString(member)}\r\n`;
+    });
+    document.getElementById('participants-collection').innerHTML = `MEMBERS INFO:\r\n${memberString}`;
+  });
+
+  meeting.members.on('members:full', (members) => {
+    let memberString = '';
+    Object.keys(members).forEach((key) => {
+      const member = members[key];
+      memberString += `Member: ${getParticipantNameString(member.participant)}, Values: ${getParticipantValueString(member)}\r\n`;
+    });
+    document.getElementById('participants-collection').innerHTML = `MEMBERS INFO:\n${memberString}`;
+  });
+
+  meeting.members.on('members:content:update', (payload) => {
+    console.log(`who started sharing: ${payload.activeContentSharingId};`);
```

Comment:
note, if app.js was more sophisticated it would simply find the memberIds and re render those.